client/settings/general-settings-section/payment-method-setup-help.js

```diff
@@ -1,36 +0,0 @@
-import { __, sprintf } from '@wordpress/i18n';
```

Comment:
Renamed to `payment-method-capability-status-pill` and moved to `wcstripe/components`